pom.xml

```diff
@@ -66,6 +68,14 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>findbugs-maven-plugin</artifactId>
+        <configuration>
```

Comment:
* File leaks on the common path are the Medium/High Severity depending on the detector
* File leaks on Exceptional paths are low

The latter bullet does not make me happy as well